bokeh/models/tiles.py

```diff
@@ -1,59 +1,65 @@
+'''
+
+'''
 from __future__ import absolute_import
-from ..model import Model
 
-from ..core.properties import Any, Dict, Float, String, Int, Bool, Override
+from ..core.properties import Any, Bool, Dict, Float, Int, Override, String
+from ..model import Model
 
 class TileSource(Model):
-    """ A base class for all tile source types. ``TileSource`` is
-    not generally useful to instantiate on its own. In general, tile sources are used as a required input for ``TileRenderer``.
+    """ A base class for all tile source types. ``TileSource`` is not generally
+    useful to instantiate on its own. In general, tile sources are used as a
+    required input for ``TileRenderer``.
 
-     Subclasses should have these properties as well:
-     x_origin_offset = Float
-     y_origin_offset = Float
-     initial_resolution = Float
     """
 
     _args = ('url', 'tile_size', 'min_zoom', 'max_zoom', 'extra_url_vars')
 
     url = String("", help="""
-    tile service url (example: http://c.tile.openstreetmap.org/{Z}/{X}/{Y}.png)
+    Tile service url e.g., http://c.tile.openstreetmap.org/{Z}/{X}/{Y}.png
```

Comment:
This was causing a bizarre sphinx failure, cleaned up the rest of the file while I was here